common/hyper_parameter.go

```diff
@@ -3,3 +3,9 @@ package common
 import "time"
 
 const RPC_TIMEOUT = 1000 * time.Millisecond
+const HEARTBEAT_INTERVAL = 3 * time.Second
+const HEARTBEAT_MONITOR_INTERVAL = 10 * time.Second
+const BLOCK_SIZE = 64 * 1024
+const BLOCK_REPLICATION = 3
+const NETWORK_BANDWIDTH = 100 * 1024 * 1024 // 100MB/s
+const READ_BLOCK_TIMEOUT = time.Duration(BLOCK_SIZE/NETWORK_BANDWIDTH)*time.Second + RPC_TIMEOUT
```

Comment:
I would suggest multiply the first term by a small constant, e.g., 3, because the network bandwidth may be shared among several flows.